core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -4903,6 +4903,20 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
             {
                 wasm_interp_call_func_native(module, exec_env, cur_func,
                                              prev_frame);
+#if WASM_ENABLE_GC != 0
```

Comment:
There may be several places calling wasm_interp_call_func_native, is it better to apply the changes in it? And had better set frame_ref only when the exception isn't thrown?

Reply:
Yes, updated.
But seems there is some issue with fast interpreter, I'm not sure if I calculate the right frame_ref address for fast interpreter

Reply:
Seems right, `frame->frame_ref` in fast interpreter isn't changed after initialized, if the ret_offset is the offset to the `frame->lp`, then it should be right.